api/src/test/java/com/codeforcommunity/rest/ApiRouterTest.java

```diff
@@ -2,122 +2,101 @@
 
 import io.vertx.core.Vertx;
 import io.vertx.core.http.*;
-import io.vertx.ext.web.*;
-import io.vertx.ext.unit.TestContext;
 import io.vertx.ext.unit.junit.VertxUnitRunner;
-import io.vertx.ext.web.RoutingContext;
+
+import static org.junit.Assert.assertEquals;
+
+import com.codeforcommunity.api.IAuthProcessor;
+import com.codeforcommunity.api.INotesProcessor;
+import com.codeforcommunity.auth.JWTAuthorizer;
 
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
 import org.mockito.junit.jupiter.MockitoExtension;
 import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.Mock;
+import org.mockito.Mockito;
 
 @RunWith(VertxUnitRunner.class)
 @ExtendWith(MockitoExtension.class)
 public class ApiRouterTest {
-  Vertx vertx;
-  HttpServer server;
-  Router router;
+  INotesProcessor myINotesProcessor = Mockito.mock(INotesProcessor.class);
+  IAuthProcessor myIAuthProcessor = Mockito.mock(IAuthProcessor.class);
+  JWTAuthorizer myJWTAuthorizer = Mockito.mock(JWTAuthorizer.class);
```

Comment:
This isn't something you have to change,  but if you want to you can statically import the `mockito.Mock` method (like `import static org.mockito.Mock`, which might make it slightly easier to mock classes. All you have to do at that point is `mock(<class>)`.